.github/actions/gcp-auth/action.yaml

```diff
@@ -0,0 +1,66 @@
+name: Authenticate against GAR
+description: |
+  Authenticates against GCP using OIDC. This action defines a default Service-Account as
+  identity.
+inputs:
+  gcp_service_account:
+    description: The Service-Account to impersonate
+    required: false
+    default: ocm-ci-opensource@gardener-project.iam.gserviceaccount.com
```

Comment:
this one is only used within ocm org, rather use `ci-gcr-operator@gardener-project.iam.gserviceaccount.com` as default value

Reply:
hmm, valid point. Maybe even better yet, we might create a dedicated service-account just for OIDC / GHAs